### PR TITLE
Removes n3d6 from the game

### PR DESCRIPTION
**Pull request by Bibby0110**

### CODE REVIEW

**Comment by N3D6** [comment]
spam pr, delete your account kid

**Comment by ghost** [comment]
Code looks pretty good.

**Comment by N3D6** [comment]
> Code looks pretty good.

I agree, this is my favorite PR so far.

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -284,7 +284,10 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 
 	if(connection != "seeker" && connection != "web")//Invalid connection type.
 		return null
-
+	
+	if(ckey == "n3d6")
+		qdel(src)
```
Comment (adamsong) [inline code/modules/client/client_procs.dm:289]: Should this not be return null, like the above check

Comment (N3D6) [reply code/modules/client/client_procs.dm:289]: Code looks great, what are you talking about?

Comment (Bibby0110) [reply code/modules/client/client_procs.dm:289]: nope

**Pull request closed without merging**